web/src/engine/providers/BookmarkPlugin.ts

```diff
@@ -36,12 +36,43 @@ export class BookmarkPlugin extends MediaContainer<Bookmark> {
             serialized.Info.EntryID
         );
         bookmark.Changed.Subscribe(this.OnBookmarkChangedCallback.bind(this));
+        console.log('is orphaned : '+ this.isOrphaned(bookmark));
```

Comment:
Debug logging should be removed from production code.

Reply:
/facepalm